models/fivetran_platform__connector_daily_events.sql

```diff
@@ -14,7 +14,7 @@ log_events as (
         connector_id,
         cast( {{ dbt.date_trunc('day', 'created_at') }} as date) as date_day,
         event_subtype,
-        message_data
+        replace(message_data, 'totalQueries', 'total_queries') as message_data
```

Comment:
Good question. My reason was thinking a staging model was a view, so downstream models might be needlessly performing this operation. However, I double checked and staging are tables not views. In light of that--should I move to the staging model or leave as-is do you think? @fivetran-joemarkiewicz

Reply:
Good thought in not wanting downstream queries to be needlessly performing these operations if the staging modes are views. Truthfully, we should have the staging models be views instead of tables. Instead of making the above change, let's keep your approach. In addition, can you create a FR for us to update the materialization of the staging models in a future breaking change update. Since we want to get this PR out as soon as possible I would prefer we don't include that update now, but it will be good for us to consider in an upcoming release.